### PR TITLE
Add AnimatedIcon to Combobox.

### PR DESCRIPTION
**Pull request by StephenLPeters**
Switch ComboBox's drop down chevron to the same animated icon as drop down button.

### CODE REVIEW

File: dev/AnimatedIcon/AnimatedIcon.cpp [hunk 1]
```diff
@@ -165,12 +165,11 @@ winrt::Size AnimatedIcon::ArrangeOverride(winrt::Size const& finalSize)
...
         };
         const auto offset = (finalSize - (visualSize * scale)) / 2;
-        const auto z = 0.0F;
         const auto rootVisual = visual.RootVisual();
-        rootVisual.Offset({ offset, z });
+        rootVisual.Offset({ offset, 0.0f });
         rootVisual.Size(arrangedSize);
-        rootVisual.Scale({ scale, z });
-        return finalSize;
+        rootVisual.Scale({ scale, 1.0f });
```
Comment (ranjeshj) [inline dev/AnimatedIcon/AnimatedIcon.cpp:171]: This seems unrelated to the change ? Interntional ?

Comment (StephenLPeters) [reply dev/AnimatedIcon/AnimatedIcon.cpp:171]: Intentional but not needed here. I could break it out to a separate change if you'd like

File: dev/ComboBox/ComboBox_themeresources.xaml [hunk 2]
```diff
@@ -604,7 +611,7 @@
                         </Grid.RowDefinitions>
                         <Grid.ColumnDefinitions>
                             <ColumnDefinition Width="*" />
-                            <ColumnDefinition Width="32" />
+                            <ColumnDefinition Width="38" />
```
Comment (beervoley) [inline dev/ComboBox/ComboBox_themeresources.xaml:614]: Shouldn't it be 40? -4 margin was changed to 4 so it give +8 size difference?

**Comment by beervoley** [comment]
@StephenLPeters can you add screenshots of ComboBox: rest + hover over dropdown button

**Review by ranjeshj — APPROVED** [review]
:shipit:

File: dev/ComboBox/ComboBox_themeresources.xaml (hunk 2 above)
Comment (StephenLPeters) [reply dev/ComboBox/ComboBox_themeresources.xaml:614]: Yes, but it was actually mispositioned by 2 pixels before this change, which there is another internal issue for.

**Review by beervoley — APPROVED** [review]
